mapie/tests/test_calibration.py

```diff
@@ -461,3 +473,20 @@ def test_pipeline_compatibility() -> None:
     mapie = MapieCalibrator(estimator=pipe)
     mapie.fit(X, y)
     mapie.predict(X)
+
+
+def test_fit_parameters_passing() -> None:
+    """
+    Test passing fit parameters, here early stopping at iteration 3.
+    Checks that underlying GradientBoosting estimators have used 3 iterations
+    only during boosting, instead of default value for n_estimators (=100).
+    """
+    mapie = MapieCalibrator(estimator=gb)
```

Comment:
```suggestion
    mapie = MapieCalibrator(estimator=gb, random_state=random_state)
```

Reply:
@thibaultcordier it seems that there is no `random_state` argument in the init function of `MapieCalibrator` so I did not make this change.